README.md

```diff
@@ -1 +1,37 @@
-# Hello-Microverse
\ No newline at end of file
+# Hello-Microverse
+> Description the project.
+- A simple webpage built using best practices, including linters.
+- Github workflow has been implemented.
+
+## Built With
+
+- HTML, CSS
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+## Authors
+
+👤 **Author**
+
+- Shudufhadzo Khorombi
+- GitHub: [@githubhandle](https://github.com/ShuduK)
+- Twitter: [@twitterhandle](https://twitter.com/KhorombiShudu)
+- LinkedIn: [LinkedIn](https://za.linkedin.com/in/shudufhadzo-khorombi-a87023134?trk=pub-pbmap)
```

Comment:
@ShuduK, I see you have updated the links to your handles but you have not updated the handles. You can use something in the lines of [@ShuduK](https://github.com/ShuduK)  to create a link to your github account.